wicket-orientdb/src/main/java/ru/ydn/wicket/wicketorientdb/model/ODocumentLinksQueryDataProvider.java

```diff
@@ -18,7 +19,7 @@ public ODocumentLinksQueryDataProvider(IModel<ODocument> docModel, IModel<OPrope
 	}
 	
 	public ODocumentLinksQueryDataProvider(IModel<ODocument> docModel, OProperty property) {
-		super("select expand("+property.getName()+") from "+property.getOwnerClass().getName()+" where @rid = :doc");
+		super("select from (select expand("+property.getName()+") from "+property.getOwnerClass().getName()+" where @rid = :doc)");
```

Comment:
Is it possible to have it dynamically build/wrapped? For just common listing double-select is not rational